cmd/zetaclientd/start.go

```diff
@@ -221,6 +232,15 @@ func start(_ *cobra.Command, _ []string) error {
 		masterLogger.Info().Msg("TSS listener received an action to shutdown zetaclientd.")
 		signalChannel <- syscall.SIGTERM
 	})
+	// debug: printout connected peers
+	go func() {
+		for {
+			time.Sleep(10 * time.Second)
+			ps := server.GetKnownPeers()
+			metrics.NumConnectedPeers.Set(float64(len(ps)))
+			telemetryServer.SetConnectedPeers(ps)
+		}
+	}()
```

Comment:
nit/question: just to clarify, since we're disabling DHT the known peers shouldn't grow beyond the initial set of peers, right?